docs/experimental-organization.md

```diff
@@ -3,38 +3,163 @@
 ## Projects
 
 ### What are projects ?
+A project is the higher level of organisation in PHIS.
+A single experimentation can include several experiments from various experimental campaigns.
 
 ### What are project properties ?
+Projects fact sheets are all public, i.e. visible for every PHIS user, including those unrelated to them.
+However, experiments and objects included in a project are accessible only to the users defined as owner or guest of this project.
```

Comment:
Rights are on experiments, not project.
Maybe start of proposal : However, experimental data included in a project are accessible only to users who belong to an owner or guest groups of the experimentation.